features/admin.applications.v1/components/forms/general-details-form.tsx

```diff
@@ -251,6 +254,43 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
         }
     };
 
+    /**
+     * Access URL may have placeholders like `${UserTenantHint}`, or `${organizationIdHint}`.
+     * This function validates the Access URL after removing those placeholders.
+     *
+     * @param value - Access URL to be validated.
+     * @returns Error message.
+     */
+    const validateAccessURL = (value: string): string => {
+        let moderatedValue: string = value?.trim();
+        let errorMsg: string;
+
+
+        let placeholdersPattern: string = "";
+
+        ApplicationManagementConstants.FORM_FIELD_CONSTRAINTS.ACCESS_URL_ALLOWED_PLACEHOLDERS.forEach(
+            (placeholder: string, index: number) => {
+                if (index == 0) {
+                    placeholdersPattern += placeholder;
+                } else {
+                    placeholdersPattern += `|${placeholder}`;
+                }
+            }
+        );
```

Comment:
Instead of building the placeholder pattern in every function call, shall we memoize the placeholderPattern using useMemo hook?

eg:

```js
const placeHoldersPattern: string = useMemo( () => {
    let placeholdersPattern: string = "";
     ApplicationManagementConstants.FORM_FIELD_CONSTRAINTS.ACCESS_URL_ALLOWED_PLACEHOLDERS.forEach(
            (placeholder: string, index: number) => {
                if (index == 0) {
                    placeholdersPattern += placeholder;
                } else {
                    placeholdersPattern += `|${placeholder}`;
                }
            }
        );

    return placeholdersPattern;
},[])
```

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/6053/commits/09fb848275f4e9556b26ffe4ebcd8ac51bc67219